builders/deploy.yaml

```diff
@@ -152,3 +152,29 @@ steps:
     - --allow-unauthenticated
     - --service-account=download-metrics@${PROJECT_ID}.iam.gserviceaccount.com
     - --set-env-vars=PROJECT_ID=${PROJECT_ID},KEY_SERVER_URL=${_KEY_SERVER_URL}
+- name: 'gcr.io/cloud-builders/gcloud'
+  waitFor: ['-']
+  args:
+    - functions
+    - deploy
+    - EfgsDownloadBatch
+    - --source=.
+    - --trigger-topic=efgs-download-batch
+    - --region=europe-west1
+    - --runtime=go113
+    - --memory=128
+    - --service-account=efgs-download-batch@${PROJECT_ID}.iam.gserviceaccount.com
+    - --set-env-vars=PROJECT_ID=${PROJECT_ID},MAX_UPLOAD_KEYS=${_MAX_UPLOAD_KEYS},EFGS_LOCAL=true
+- name: 'gcr.io/cloud-builders/gcloud'
+  waitFor: ['-']
+  args:
+    - functions
+    - deploy
+    - EfgsDownloadBatch
```

Comment:
two functions with the same name but different trigger? I don't think this is possible to do.

Reply:
It's a mistake.